atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbKvs.java

```diff
@@ -795,7 +829,17 @@ public Void apply(DbDdlTable table) {
     private <T> T runDdl(TableReference tableRef, Function<DbDdlTable, T> runner) {
         ConnectionSupplier conns = new ConnectionSupplier(connections);
         try {
-            return runner.apply(dbTables.createDdl(tableRef.getQualifiedName(), conns));
+            /* The ddl actions can used both the fully qualified name and the internal name */
+            return runner.apply(dbTables.createDdl(tableRef, conns));
+        } finally {
+            conns.close();
```

Comment:
probably worth a separate PR, but can we use try-with-resources for these to get proper suppressed exceptions?